config/intersphinx.yaml

```diff
@@ -3,6 +3,6 @@ url: https://docs.python.org/2/
 path: python2.inv
 ---
 name: mongodb
-url: https://docs.mongodb.com/manual/
+url: https://mongodb.com/docs/manual/
```

Comment:
Issue: Could everything in the config directory except config/redirects be deleted? I believe this is appropriate as the repo is on next-gen publishing. 

Excerpt of [relevant blog post](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=109644393):

>  Delete everything in the config/ directory except config/redirects , which is still used for redirects . Delete conf.py . Delete the Makefile . Shudder in delight at all the deleting.